packages/geo/src/bounds.ts

```diff
@@ -66,17 +68,41 @@ export class Bounds {
         return { x: this.x, y: this.y, width: this.width, height: this.height };
     }
 
+    /**
+     * Convert to Mercator.BBox
+     */
+    public toBbox(): Mercator.BBox {
```

Comment:
Maybe use GeoJSON.BBox here instead. It is 3D as Mercator.BBox is only 2D a typescript ripple affect if I change it